app/controllers/api/mentor_discussion_posts_controller.rb

```diff
@@ -3,24 +3,17 @@ class MentorDiscussionPostsController < BaseController
     before_action :use_mentor_discussion, only: %i[index create]
 
     def index
-      posts = @discussion.
-        posts.
-        joins(:iteration).
-        where(iterations: { idx: params[:iteration_idx] })
```

Comment:
Nice.

I take it there is no deleted flag?

Reply:
No, not for now. But this is something we should think about, as there is in v2 so we'll need to handle this. Good thinking.